apis/v1alpha2/grpcroute_types.go

```diff
@@ -391,10 +392,31 @@ type GRPCHeaderMatch struct {
 	Value string `json:"value"`
 }
 
+// GRPCHeaderMatchType specifies the semantics of how GRPC header values should
+// be compared. Valid GRPCHeaderMatchType values are:
+//
+// * "Exact" * "RegularExpression"
```

Comment:
I think this will need this formatting to render correctly?
```suggestion
// * "Exact"
// * "RegularExpression"
```

Also, there's no clarification about regex support - elsewhere we have it as ImplementationSpecific because of flavor choices, should we replicate that here?

Reply:
Good catch! This looks like a broader issue, filed https://github.com/kubernetes-sigs/gateway-api/issues/1581 to follow up.